exports.js

```diff
@@ -796,6 +796,7 @@ module.exports = {
       
         'bucketLoggingEnabled'          : require(__dirname + '/plugins/alibaba/oss/bucketLoggingEnabled.js'),
         'ossBucketPrivate'              : require(__dirname + '/plugins/alibaba/oss/ossBucketPrivate.js'),
+        'ossBucketTransferAcceleration' : require(__dirname + '/plugins/alibaba/oss/ossBucketTransferAcceleration.js'),
```

Comment:
```suggestion
        'BucketTransferAcceleration'    : require(__dirname + '/plugins/alibaba/oss/BucketTransferAcceleration.js'),
```

Reply:
According to the naming convention, plugin names do not start with a capital letter. 'bucketTransferAcceleration' pulgin already exists. Are you sure to rename as you suggested?